spec/factories/application.rb

```diff
@@ -31,13 +31,13 @@
         "student0_application_code_samples": FFaker::Lorem.paragraph,
         "student0_application_learning_history": FFaker::Lorem.paragraph,
         "student0_application_skills": FFaker::Lorem.paragraph,
-        "student0_application_code_background": FFaker::Lorem.paragraph,
+        "student0_application_language_learning_period": FFaker::Lorem.paragraph,
```

Comment:
From what I remember, `learning-period` is a string of numbers, e.g. `"1-3"`, etc. I'd suggest to also keep it like this in the Factories, e.g. using something like
```ruby
User::MONTH_LEARNING.sample
```
It may be personal preference, but I always tend to think, having ones test data as close to the *"real world"* as feasable without too much effort, helps keeping the test suite sane and readable *(and hopefully powerful enough to find out weirdnesses)*